bundle/edu.gemini.shared.util/src/main/java/edu/gemini/shared/util/immutable/Either.java

```diff
@@ -0,0 +1,80 @@
+package edu.gemini.shared.util.immutable;
+
+import java.io.Serializable;
+
+/**
+ * Simulates a Scala Either[S,T].
+ */
+public final class Either<L,R> implements Serializable {
```

Comment:
I'm really, really not crazy about extending `shared.util.immutable` but on the other hand I can see why we might need this.